src/test/kotlin/org/rust/ide/intentions/ImportNameIntentionStdTest.kt

```diff
@@ -0,0 +1,59 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.intentions
+
+import com.intellij.testFramework.LightProjectDescriptor
+
+class ImportNameIntentionStdTest : RsIntentionTestBase(ImportNameIntention()) {
+
+    override fun getProjectDescriptor(): LightProjectDescriptor = WithStdlibRustProjectDescriptor
+
+    fun `test import struct from std crate`() = doAvailableTest("""
+        fn foo<T: Read/*caret*/>(t: T) {}
+    """, """
+        use std::io::Read;
+
+        fn foo<T: Read/*caret*/>(t: T) {}
+    """, ImportNameIntention.Testmarks.autoInjectedCrate)
+
+    // TODO: change this test after reexports support
+    fun `test import struct from not std crate`() = doAvailableTest("""
```

Comment:
Is there a good way to create tests with some extern crates (not from stdlib)?

Reply:
This could be handled via custom project descriptor, see `WithStdlibAndDependencyRustProjectDescriptor` for example.

Reply:
Oh, thanks)
Then I'll change these tests to avoid unnecessary changes in future when we will support reexports in auto import